server/configuration.go

```diff
@@ -33,6 +33,17 @@ type configuration struct {
 	JitsiNamingScheme  string
 }
 
+const publicJitsiServerUrl = "https://meet.jit.si"
+
+// GetJitsiURL return the currently configured JitsiURL or the URL from the
+// public servers provided by Jitsi.
+func (c *configuration) GetJitsiURL() string {
+	if len(c.JitsiURL) > 0 {
+		return c.JitsiURL
+	}
+	return publicJitsiServerUrl
```

Comment:
That seams like a surprising behaviour to me. If an admin puts an empty string in the config field, I would expect the plugin to just not work. Using another URL is that case it not `Obvious` to me.

@jespino @jasonblais Thoughts?

Reply:
Is the experience here that we override the setting if left empty? Is there a use case for when an admin wants to leave an empty server URL? 

The target use case for me is when you first install & configure Jitsi. By setting the default Jitsi server to meet.jit.si, an admin simply has to install and enable the plugin to try it out.

Reply:
I don't think is confusing, there is not reason to have a system without URL but with jitsi enabled. I think can be clearly explain in the help text to avoid surprises, but other than that, I don't think is dangerous.

Reply:
I don't want to argue against having meet.jit.si as the default. If the plugin is freshly installed, the URL should be `meet.jit.sit`.

I also can't think of a reason an admin would leave the setting empty. But if (s)he does by accident, one would expect that the plugin won't work any longer. 

To me a default value is the first value that is used on a fresh install. If that value is ever changes, the new one is used instead. Which is this case is an empty string.

Reply:
@jasonblais what do you think?

Reply:
I disagree.

If an admin removes the URL by accident, and we break the plugin, we are [not forgiving](https://docs.mattermost.com/developer/fx-guidelines.html#fast-obvious-forgiving).

We have many System Console settings such as [max users per team](https://docs.mattermost.com/administration/config-settings.html#max-users-per-team) where if the value is removed, it reverts back to the default. This is to prevent the system from breaking due to an admin making a mistake.

I see two options: If we believe the primary use case for leaving the setting empty is:
1. by accident, we should fall back to the default, `meet.jit.si`.
2. intentional because they want to disable the plugin, then we should disable the plugin.

These are more forgiving (and to me more obvious) experiences. 

My vote is for 1). If they wanted to disable the plugin, they would be more likely to disable it than removing the URL.

Reply:
^One case that comes to mind: if reverting to a public Jitsi service poses a privacy/security concern, then option 1 may not be most forgiving and we should go with option 2. However, I don't have strong knowledge on that to know.

Reply:
Then lets ask @jupenur about it. What do you think about it? is something that is completly wrong, or is something that is an acceptable compromise while is well documented and described?

Reply:
Hmm... [MM-25859](https://mattermost.atlassian.net/browse/MM-25859) might be a concern, specifically if we update the documentation now, but an existing user doesn't notice the documentation change or realize the defaults have changed as they update the plugin.

I'm not familiar with the validation logic available in the System Console, but could we just not allow an empty string at all?

Reply:
We can disallow an empty string, and try to default to a https://meet.jit.si, but then we will have the same problem with the previous users that have that settings to blank.

Reply:
Discussed with @jespino, proposal is to add client-side validation in the System Console when an admin removes the default URL, leaves the field blank and attempts to save the setting.

We can use the text `"Jitsi Server URL" is required.` which is consistent with other similar System Console settings

Reply:
The plugins, by default doesn't allow to validate or verify anything from the plugin side, so we would need a more rich (custom) configuration screen for jitsi. I going to create another issue for 2.1.0 version.

Reply:
Created here #110